### PR TITLE
Use template literals instead of concatenation

### PR DESCRIPTION
**Pull request by rohit2sharma95**
Replace string concatenation with template literals.

### CODE REVIEW

File: js/src/popover.js [hunk 1]
```diff
@@ -28,11 +28,11 @@ const Default = {
   offset: [0, 8],
   trigger: 'click',
   content: '',
-  template: '<div class="popover" role="tooltip">' +
-              '<div class="popover-arrow"></div>' +
-                '<h3 class="popover-header"></h3>' +
-              '<div class="popover-body"></div>' +
-            '</div>'
+  template: `<div class="popover" role="tooltip">
```
Comment (XhmikosR) [inline js/src/popover.js:31]: This when minified will have the empty spaces...

File: js/src/collapse.js [hunk 2]
```diff
@@ -72,8 +72,7 @@ class Collapse extends BaseComponent {
     this._isTransitioning = false
     this._config = this._getConfig(config)
     this._triggerArray = SelectorEngine.find(
-      `${SELECTOR_DATA_TOGGLE}[href="#${this._element.id}"],` +
-      `${SELECTOR_DATA_TOGGLE}[data-bs-target="#${this._element.id}"]`
+      `${SELECTOR_DATA_TOGGLE}[href="#${this._element.id}"],${SELECTOR_DATA_TOGGLE}[data-bs-target="#${this._element.id}"]`
```
Comment (XhmikosR) [inline js/src/collapse.js:75]: Not sure this makes sense since it's less readable plus the minifier should already join these.

Comment (rohit2sharma95) [reply js/src/collapse.js:75]: True, less readable now.

**Comment by XhmikosR** [comment]
For what is worth, we don't enforce template literals. We did at some point, but it seems it's not enforced by xo now.

**Review by XhmikosR — APPROVED** [review]

**Pull request merged**